acnportal/algorithms/sorted_algorithms.py

```diff
@@ -183,6 +183,17 @@ def first_come_first_served(evs, iface):
     return sorted(evs, key=lambda x: x.arrival)
 
 
+def last_come_first_served(evs, iface):
```

Comment:
We require a specific call signature for the `sort_fn`'s of (List[EV], Interface) -> List[EV], even when the interface isn't required. Do we want to mention this call signature in the docs?

Reply:
I added a note in each function where iface is not used.